src/bin/prompts.js

```diff
@@ -1,51 +1,78 @@
 const {prompt} = require('prompts');
-const {GAME_NAMES, PLATFORM_NAMES} = require('../const');
+const {GAME_NAMES, GAME_ARGS, PLATFORM_NAMES, PLATFORM_ARGS} = require('../const');
+const {Signale} = require('signale');
+
+const statusLog = new Signale({interactive: true});
+const argv = require('minimist')(process.argv.slice(2));
 
 async function promptCredentials() {
-  const {email, password} = await prompt([
-    {
-      type: 'text',
-      name: 'email',
-      message: 'Shift Email'
-    },
-    {
-      type: 'password',
-      name: 'password',
-      message: 'Shift Password'
-    }
-  ]);
+  if ( argv['email'] && argv['password'] ) {
+      var email = argv['email'];
+      var password = argv['password']; 
+  } else {
+    statusLog.success('Login to your SHiFT account:');
+
+    var {email, password} = await prompt([
+      {
+        type: 'text',
+        name: 'email',
+        message: 'Shift Email'
+      },
+      {
+        type: 'password',
+        name: 'password',
+        message: 'Shift Password'
+      }
+    ]);
+  }
 
   return {email, password};
 }
 
 async function promptGamePlatform() {
-  const {platform, game} = await prompt([
-    {
-      type: 'select',
-      name: 'platform',
-      message: 'Platform',
-      choices: PLATFORM_NAMES.map((title, value) => ({title, value}))
-    },
-    {
-      type: 'select',
-      name: 'game',
-      message: 'Game',
-      choices: GAME_NAMES.map((title, value) => ({title, value}))
-    }
-  ]);
+  if ( argv['platform'] ) {
+    var platform = PLATFORM_ARGS.indexOf(argv['platform']);
+  } else {
+    var {platform} = await prompt([
+      {
+        type: 'select',
+        name: 'platform',
+        message: 'Platform',
+        choices: PLATFORM_NAMES.map((title, value) => ({title, value}))
+      }
+    ]);
+  }
+  
+  if ( argv['games'] ) {
```

Comment:
Can do the same split up with this function, `promptGame` `promptPlatform`

Reply:
Yeah, that would probably be better than my code.